src/support.h

```diff
@@ -29,16 +32,44 @@ class AreaSupport {
      */
     static void generateSupportAreas(SliceDataStorage& storage, unsigned int mesh_idx, unsigned int layer_count, std::vector<Polygons>& supportAreas);
 
+    /*!
+     * Generate support bottom areas for a given mesh.
+     *
```

Comment:
> These functions also subtract the interface from the support areas, right?

Yes, both the support roof and support bottom subtract their respective parts from the support areas. I've documented this now. This happens in `generateSupportInterfaceLayer`.

I've worded it differently so that it's now clear that the support roof and bottom are separated from the normal support areas.